rust-peer/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+# https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file

```

Comment:
maybe add a comment similar to what you mentioned here:

> note that ipv6.is_global() is https://github.com/rust-lang/rust/issues/27709 on [nightly](https://doc.rust-lang.org/std/net/enum.IpAddr.html#method.is_global), I hope that's not a problem? I added a toolchain config for that. If not we can code around it with logic I suppose...

Reply:
Added.

Reply:
Actually, I think we'll just use `!ip.is_loopback()` so we don't have to set the toolchain at all. All we really want to do is make sure we're not adding a local IP anyway, so it works